benchmark/cluster/vgg16/README.md

```diff
@@ -62,6 +62,28 @@ Setting environment variable: `MKL_NUM_THREADS=1`.
 *The performance gap between Fuild and v2 comes from the network interference.*
 
 
+## Test Result (GPU with flowers)
```

Comment:
Maybe change "flowers" to "flowers dataset", btw does which dataset we use matters in the performance benchmark?